lib/install.js

```diff
@@ -350,40 +341,39 @@ function install (gyp, argv, callback) {
         })
       }
 
+      //TODO: download appropriate node.lib
       function downloadNodeLib (done) {
-        log.verbose('on Windows; need to download `node.lib`...')
+        log.verbose('on Windows; need to download `' + release.name + '.lib`...')
         var dir32 = path.resolve(devDir, 'ia32')
           , dir64 = path.resolve(devDir, 'x64')
-          , nodeLibPath32 = path.resolve(dir32, 'node.lib')
-          , nodeLibPath64 = path.resolve(dir64, 'node.lib')
-          , nodeLibUrl32 = distUrl + '/v' + version + '/node.lib'
-          , nodeLibUrl64 = distUrl + '/v' + version + '/x64/node.lib'
+          , libPath32 = path.resolve(dir32, release.name + '.lib')
```

Comment:
we have local `libPath*`'s and also `release.libPath*`'s, the former are where we put the x.lib files for the compiler to use (`ia32` and `x64` directories) and the latter is their location on the server and in the shasums lookup